catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Registry.view.js

```diff
@@ -208,7 +216,44 @@ define([
             }
         });
 
+        RegistryView.DeleteModal = Marionette.ItemView.extend({
+            template: 'deleteNodeModal',
+            className: 'modal',
+            events: {
+                'click .submit-button' : 'deleteNode',
+                'click .cancel-button' : 'cancel',
+                'click .close': 'cancel'
+            },
+            deleteNode: function() {
+                 this.model.collection.deleteNodes([this.model.get('id')]);
+                 this.close();
+            },
+            cancel: function() {
+                this.close();
+            },
+            close: function() {
+                this.$el.off('hidden.bs.modal');
+                this.$el.off('shown.bs.modal');
+                this.$el.modal("hide");
+            },
+            serializeData: function() {
+                var data = {};
+                if (this.model) {
+                    data = this.model.toJSON();
+                }
+                data.name = this.getNodeName();
+                return data;
+           },
+           getNodeName: function() {
+                var nodeName;
+                this.model.get("RegistryObjectList").ExtrinsicObject.forEach(function(extObj) {
```

Comment:
You could replace this with `this.model.getObjectOfType('urn:registry:federation:node').Name`